python/hail/tests.py

```diff
@@ -37,6 +37,88 @@ def convert_struct_to_dict(x):
     else:
         return x
 
+def create_all_values_datasets():
+    all_values = hl.struct(
+        f32=hl.float32(3.14),
+        i64=hl.int64(-9),
+        m=hl.null(hl.tfloat64),
+        astruct=hl.struct(a=5, b=5.5),
```

Comment:
do we need empty structs, sets, tuples, etc?

Reply:
I added a variety of missing aggregate types.  I didn't really mean to make this exhaustive.  (We can always go back to this version and add more stuff to the files.)